tests/api/runtime_handlers/test_sparkjob.py

```diff
@@ -257,38 +236,17 @@ async def test_delete_resources_with_force(self, db: Session, client: TestClient
             self.runtime_handler,
             len(list_namespaced_crds_calls),
         )
-        self._assert_list_namespaced_pods_calls(
-            self.runtime_handler,
-            len(list_namespaced_pods_calls),
-            self.pod_label_selector,
-        )
         self._assert_run_reached_state(
             db, self.project, self.run_uid, RunStates.running
         )
-        await self._assert_run_logs(
-            db,
-            self.project,
-            self.run_uid,
-            log,
-            self.driver_pod.metadata.name,
-        )
 
-    @pytest.mark.asyncio
-    async def test_monitor_run_completed_crd(self, db: Session, client: TestClient):
+    def test_monitor_run_completed_crd(self, db: Session, client: TestClient):
         list_namespaced_crds_calls = [
             [self.running_crd_dict],
             [self.completed_crd_dict],
         ]
         self._mock_list_namespaced_crds(list_namespaced_crds_calls)
-        # for the get_logger_pods with proper selector
-        list_namespaced_pods_calls = [
```

Comment:
I believe this test should call list pods at least once for the state threshold check. since you removed the assertion below I think it removes coverage